ideainspect.groovy

```diff
@@ -242,6 +261,59 @@ private File findIdeaExecutable(OptionAccessor cliOpts) {
   ideaExecutable
 }
 
+private modifyIdeaProps(OptionAccessor cliOpts){
+  println "#"
```

Comment:
Oh. It seems I missed the point: My idea was to append/add those line in the updated `idea.properties` file. That's why I also mentioned the _GENERATED START_ and _END_ stanzas: So you could eaisly erase/remove the block on cleanup.